dev/com.ibm.ws.sipcontainer/src/com/ibm/ws/sip/container/servlets/SipApplicationSessionImpl.java

```diff
@@ -669,23 +669,28 @@ protected List<IBMSipSession> getAllSIPSessions(boolean create)
 				}
 				return Collections.EMPTY_LIST;
 			}
-			int size = m_transactionUsers.size();
+			
+			else {
+				int size = m_transactionUsers.size();
 
 			//Moti: we create new ArrayList to prevent later any modification
 			// to the iterator on that list.
-			List<IBMSipSession> result = new ArrayList<IBMSipSession>(size); //approximation only.
-			synchronized (m_transactionUsers) {
-				TransactionUserWrapper tu = null;
-				for (int i = 0 ; i < size ; i++) {
-					tu = m_transactionUsers.get(i);
-					result.addAll(tu.getAllSipSessions(create));
-				}
-			}
-			if (c_logger.isTraceDebugEnabled()) {
-				c_logger.traceDebug(this, "getAllSIPSessions", "found SIP sessions. count:"+result.size());
-			}
-
+			
+				List<IBMSipSession> result = new ArrayList<IBMSipSession>(size); //approximation only.
+					synchronized (m_transactionUsers) {
+						TransactionUserWrapper tu = null;
+						for (int i = 0 ; i < size ; i++) {
+						tu = m_transactionUsers.get(i);
+						result.addAll(tu.getAllSipSessions(create));
+						}
+			
+						if (c_logger.isTraceDebugEnabled()) {
+							c_logger.traceDebug(this, "getAllSIPSessions", "found SIP sessions. count:"+result.size());
+						}
+					}
 			return result;
+			}
+			
```

Comment:
Just a general comment: could you fix the indentation / formatting here and a few other places as some of it a bit hard to follow. Which brackets match.